lab-server/.env.example

```diff
@@ -1,8 +1,8 @@
 # Note: the following env example is configured so that when following the
 # contribution guide for a LOCAL SETUP no values need to be changed (apart from
-# the keycloak lab-server client ID which is generated when creating the client
-# in the Keycloak admin panel). This environment will NOT work out of the box
-# when trying to deploy these services. Refer to the docker-compose file when
+# the keycloak lab-server scret which is generated when creating the client in
```

Comment:
```suggestion
# the keycloak lab-server secret which is generated when creating the client in
```

Reply:
Fixed in [a011e07](https://github.com/hpi-dhc/PharMe/pull/316/commits/a011e074815948b3dd1d09a71c1c2952b8bc8c87).